test/runtests.jl

```diff
@@ -8,6 +8,8 @@ using Random
 using SpecialFunctions
 using Test
 using Flux
+
+using Pkg; Pkg.add(url="https://github.com/JuliaDiff/ForwardDiff.jl", rev="nansafe")
```

Comment:
Apparently
```suggestion
using Pkg
Pkg.add(PackageSpec(; url="https://github.com/JuliaDiff/ForwardDiff.jl", rev="nansafe"))
```
is needed in older Julia versions.